src/gluttony/core.clj

```diff
@@ -108,11 +108,23 @@
                                   :long-polling-duration long-polling-duration
                                   :exceptional-poll-delay-ms exceptional-poll-delay-ms
                                   :heartbeat (:heartbeat opts)
-                                  :heartbeat-timeout (:heartbeat-timeout opts)})]
+                                  :heartbeat-timeout (:heartbeat-timeout opts)
+                                  :receiver-enabled (atom true)})]
     (p/-start consumer)))
 
 (defn stop-consumer
   "Takes a consumer created by start-consumer and stop all processes and the client if it is created
    in start-consumer. This should be called to stopped consuming messages."
   [^Consumer consumer]
   (p/-stop consumer))
+
+(defn disable-receivers
+  "Stop fetching messages from AWS SQS.
+  WARN: Receivers during long-polling are not canceled."
+  [^Consumer consumer]
+  (p/-disable-receivers consumer))
+
+(defn enable-receivers
```

Comment:
I think `enable` and `disable` does not match for public API name since there are already `start` and `stop`.

I want to use `stop-receivers` and `start-receivers` as public API and they fit for docstring you wrote.